improver/calibration/ensemble_calibration.py

```diff
@@ -327,16 +330,17 @@ def calculate_normal_crps(
 
         """
         if predictor.lower() == "mean":
-            beta = initial_guess[2:]
+            alpha_beta = initial_guess[:2]
         elif predictor.lower() == "realizations":
-            beta = np.array(
-                [initial_guess[2]] + (initial_guess[3:] ** 2).tolist(), dtype=np.float32
+            alpha_beta = np.array(
+                [initial_guess[0]] + (initial_guess[1:-2] ** 2).tolist(),
+                dtype=np.float32,
             )
 
         new_col = np.ones(truth.shape, dtype=np.float32)
         all_data = np.column_stack((new_col, forecast_predictor))
-        mu = np.dot(all_data, beta)
-        sigma = np.sqrt(initial_guess[0] ** 2 + initial_guess[1] ** 2 * forecast_var)
+        mu = np.dot(all_data, alpha_beta)
+        sigma = np.sqrt(initial_guess[-2] ** 2 + initial_guess[-1] ** 2 * forecast_var)
```

Comment:
I've made a minor modification here to split the `initial guess`, so that the variables are named when used in the equations. As [scipy.optimize.minimize](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html) expects the initial guess to be a numpy array, this means that providing a `dict` isn't possible, which I think might be nicest solution. I've had a look at using a structured numpy array, which isn't something I've used before, however, it doesn't seem possible to compare two numpy structured arrays. For example, the following code will generate a structured numpy array of, what I think seems like the correct format, however, the comparison fails. This comparison fails for situations other than this simplified test case.
```
def test_basic(self):
        coeff_names = ["alpha", "beta", "gamma", "delta"]
        dtype = dict(names=coeff_names,
                     formats=[np.float32] * len(coeff_names))
        expected = np.array(
            (1.0, 1.0, 0.0, 1.0), dtype=dtype
        )
        result = np.array(
            (1.0, 1.0, 0.0, 1.0), dtype=dtype
        )
        np.testing.assert_array_almost_equal(
            result, expected
        )
```
Therefore for the moment, I've only made minor edits to this.